### ISSUE TITLE
ERA001 should recognize pyrefly ignore comments

### ISSUE THREAD

**Issue by skewty** [astral-sh/ruff#28184]
### Summary

```python
# pyrefly: ignore [bad-argument-type]
task = asyncio.create_task(handler(alarm))
```
The special pyrefly ignore comment is flagged as ERA001.

Can we please, until a better algorithm / solution is found, at a minimum, support pyrefly ignore comments? Perhaps adjust the pattern to match `[a-z]+\: ignore` so all are (pyright, basedpyright, pyrefly, ty) supported?

**Comment by ntBre** [comment]
I can't reproduce this on latest Ruff: https://play.ruff.rs/a97b0f34-12a5-4c0b-8648-cb06d4eee274

Support for `pyrefly` comments should have been added in 0.12.8: https://github.com/astral-sh/ruff/commit/b95d22c08eb3a0f58da44c1403e3ab7b4921540e.
